libs/ddhub-client-gateway-message-broker/src/lib/services/ddhub-topics.service.ts

```diff
@@ -401,6 +434,7 @@ export class DdhubTopicsService extends DdhubBaseService {
               owner,
               page,
               tags,
+              includeDeleted: includeDeleted ? 'true' : 'false',
```

Comment:
@azam-ismail why message broker need a string as param for `includeDeleted`?

Reply:
that for part for support ui